torch/_inductor/fx_passes/quantization.py

```diff
@@ -656,6 +658,15 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
                 ),
                 dtype=original_pattern_output_dtype,
             ),
+            UnaryAttr("swish", [], ""): generate_pattern_with_output_quant(
+                _unary_fusion_pattern(
+                    _silu_fusion,
+                    get_dequantize_qconv_pt2e_pattern(1 if is_bf16 else 2),
+                    2,
+                    is_bf16,
+                ),
+                dtype=original_pattern_output_dtype,
```

Comment:
For `int-mixed-bf16` and `int8 output`, here we will have the `to_fp32` between `qconv->mul/sigmod` (from `silu`). I think the pattern should have `to_fp32` between `silu` and output's `quant`. Should we keep using `torch.float32` instead of `original_pattern_output_dtype` here?

Reply:
We may also check `qconv2d_unary_matcher_nodes` in UT to ensure the correct number of nodes in pattern has been matched.

Reply:
Thanks for reminder. Updated under your advise.